lib/components/app/popup.tsx

```diff
@@ -4,7 +4,12 @@ import coreUtils from '@opentripplanner/core-utils'
 import React, { useEffect } from 'react'
 
 type Props = {
-  content?: { appendLocale?: boolean; id?: string; url?: string }
+  content?: {
+    appendLocale?: boolean
+    id?: string
+    iframe?: boolean
```

Comment:
Maybe rename this to `embedded` or something?

Reply:
Good idea! I've renamed the variable